kura/distrib/src/main/resources/generic-aarch64/kura_install.sh

```diff
@@ -0,0 +1,129 @@
+systemctl enable NetworkManager
+systemctl start NetworkManager
+
+# set up users and grant permissions
+cp ${INSTALL_DIR}/kura/install/manage_kura_users.sh ${INSTALL_DIR}/kura/.data/manage_kura_users.sh
+chmod 700 ${INSTALL_DIR}/kura/.data/manage_kura_users.sh
+${INSTALL_DIR}/kura/.data/manage_kura_users.sh -i
+
+# replace snapshot_0 and iptables.init with correct interface names
+if python3 -V > /dev/null 2>&1
+then
+    python3 ${INSTALL_DIR}/kura/install/find-net-interfaces.py ${INSTALL_DIR}/kura/user/snapshots/snapshot_0.xml ${INSTALL_DIR}/kura/install/iptables.init ${INSTALL_DIR}/kura/framework/kura.properties
+else
+    echo "python3 not found. snapshot_0.xml, iptables.init, and kura.properties files may have wrong interface names. Default is eth0 and wlan0. Please correct them manually if they mismatch."
+fi
+
+# copy snapshot_0.xml
+cp ${INSTALL_DIR}/kura/user/snapshots/snapshot_0.xml ${INSTALL_DIR}/kura/.data/snapshot_0.xml
+
+# set up default firewall configuration
+cp ${INSTALL_DIR}/kura/install/iptables.init ${INSTALL_DIR}/kura/.data/iptables
+chmod 644 ${INSTALL_DIR}/kura/.data/iptables
+cp ${INSTALL_DIR}/kura/.data/iptables /etc/sysconfig/iptables
+cp ${INSTALL_DIR}/kura/install/firewall.init ${INSTALL_DIR}/kura/bin/firewall
+chmod 755 ${INSTALL_DIR}/kura/bin/firewall
+cp ${INSTALL_DIR}/kura/install/firewall.service /lib/systemd/system/firewall.service
+chmod 644 /lib/systemd/system/firewall.service
+sed -i "s|/bin/sh KURA_DIR|/bin/bash ${INSTALL_DIR}/kura|" /lib/systemd/system/firewall.service
+systemctl daemon-reload
+systemctl enable firewall
+
+# disable NTP service
+if command -v timedatectl > /dev/null ;
+  then
+    timedatectl set-ntp false
+fi
+
+# set up logrotate - no need to restart as it is a cronjob
+cp ${INSTALL_DIR}/kura/install/kura.logrotate /etc/logrotate-kura.conf
+
+if [ ! -f /etc/cron.d/logrotate-kura ]; then
+    test -d /etc/cron.d || mkdir -p /etc/cron.d
+    touch /etc/cron.d/logrotate-kura
+    echo "*/5 * * * * root /usr/sbin/logrotate --state /var/log/logrotate-kura.status /etc/logrotate-kura.conf" >> /etc/cron.d/logrotate-kura
+fi
+
+# set up systemd-tmpfiles
+cp ${INSTALL_DIR}/kura/install/kura-tmpfiles.conf /etc/tmpfiles.d/kura.conf
+
+# set up kura files permissions
+chmod 700 ${INSTALL_DIR}/kura/bin/*.sh
+chown -R kurad:kurad /opt/eclipse
+chmod -R go-rwx /opt/eclipse
+chmod a+rx /opt/eclipse
+find /opt/eclipse/kura -type d -exec chmod u+x "{}" \;
+
+keytool -genkey -alias localhost -keyalg RSA -keysize 2048 -keystore /opt/eclipse/kura/user/security/httpskeystore.ks -deststoretype pkcs12 -dname "CN=Kura, OU=Kura, O=Eclipse Foundation, L=Ottawa, S=Ontario, C=CA" -ext ku=digitalSignature,nonRepudiation,keyEncipherment,dataEncipherment,keyAgreement,keyCertSign -ext eku=serverAuth,clientAuth,codeSigning,timeStamping -validity 1000 -storepass changeit -keypass changeit
+
+bash "${INSTALL_DIR}/kura/install/install-jdk-dio-properties.sh"
+
+# customizing kura.properties
+KURA_PLATFORM=$( uname -m )
```

Comment:
You can use `${INSTALL_DIR}/kura/install/customize-installation.sh` here

Reply:
I should've done it